src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml

```diff
@@ -65,50 +63,44 @@
                                             sfa-table-sort-column="@Model.SortColumn"
                                             sfa-sort-direction="@Model.SortDirection">@Model.DateHeaderColumnNameForReviewStatus()</sfa-sortable-column-header>
-                                        </th>
-                                    </tr>
+            </th>
+        </tr>
     </thead>
     <tbody class="govuk-table__body">
         @if (Model?.PaginatedList != null)
         {
             @foreach (var application in Model.PaginatedList.Items)
             {
-        <tr class="govuk-table__row">
-            <td class="govuk-table__cell">
-                @if (application.SequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
-                {
-                    <text>Organisation</text>
-                }
-                else if (application.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
-                {
-                    <text>Standard</text>
-                }
-            </td>
-            <td class="govuk-table__cell">
-                <a class="govuk-link"
-                   asp-action="Sequence"
-                   asp-controller="Application"
-                   asp-route-applicationId="@application.ApplicationId"
-                   asp-route-sequenceNo="@application.SequenceNo"
-                   asp-route-backAction="WithdrawalApplications"
-                   asp-route-backController="WithdrawalApplication">
-                    @if (application.SequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
-                    {
-                        @application.OrganisationName
-                    }
-                    else if (application.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell">
+                        <a class="govuk-link"
+                           asp-action="Sequence"
+                           asp-controller="Application"
+                           asp-route-applicationId="@application.ApplicationId"
+                           asp-route-sequenceNo="@application.SequenceNo"
+                           asp-route-backAction="WithdrawalApplications"
+                           asp-route-backController="WithdrawalApplication">
+                            @application.OrganisationName
+                        </a>
+                    </td>
+                    <td class="govuk-table__cell">
+                        @if (application.SequenceNo == ApplyConst.ORGANISATION_WITHDRAWAL_SEQUENCE_NO)
+                        {
+                            <text>Withdrawal from register</text>
+                        }
+                        else if (application.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
+                        {
+                            @($"{application.StandardName} (ST{application.StandardCode})")
```

Comment:
Should this StandardCode - the code is different from the Reference?

Reply:
I've replaced with Standard reference. Std ref was already returned from the API as part of the ApplicationSummaryItem